reference_frame: "world_origin"

control_action_interval: 0.01 # [s]
uvdar_msg_interval: 0.1 # [s]

# offset from the leader position to be maintained
desired_offset:
  x: -4.0 # [m]
  y: 5.0 # [m]
  z: 0.0 # [m]

# rotation offset around the world vertical axis
heading_offset: 0.3 # [rad]

# Noise parameters of the Kalman velocity estimator

# Process noise
Q: [10, 0, 0, 0, 0, 0,
    0, 10, 0, 0, 0, 0,
    0, 0, 10, 0, 0, 0,
    0, 0, 0, 1, 0, 0,
    0, 0, 0, 0, 1, 0,
    0, 0, 0, 0, 0, 1]

# # Measurement noise
R: [75, 0, 0,
    0, 75, 0,
    0, 0, 75]
